src/Equinox.Cosmos/Cosmos.fs

```diff
@@ -1179,7 +1165,7 @@ type Discovery =
             UriAndKey (Uri uri, key)
         | _ -> invalidArg "connectionString" "unrecognized connection string format; must be `AccountEndpoint=https://...;AccountKey=...=;`"
 
-type EquinoxCosmosOperationsFactory
```

Comment:
You really hate the `Equinox` in front, huh? ;)

Reply:
Maybe ;) as mentioned, there;'s nothing with `Equinox` in the type name anywhere else so the removal is driven by an ill-defined consistency instinct....

Reply:
it gets worse :( 

I've pended renaming it to `StoreOperationsFactory.Create() : Equinox.Cosmos.Store.Operations`

Am going to spend some quality time with my thesaurus (and PoeSA book) as any glee you might imagine me having is not here...

Reply:
I'm coming back to thinking it should be called [something including] `Gateway` based on https://martinfowler.com/eaaCatalog/gateway.html

`Equinox.Cosmos.Store` currently has short names and its not a mainstream direct usage thing

if you don't hate the name, then I'd try to find a better name for the one that's in `.Internal` atm ?